docs/dev/docker-hub.md

```diff
@@ -348,7 +348,7 @@ The `--authorizer` command lines option allows to provide the authorizer class S
 
 JMX ScyllaDB service is initialized from the `/scylla-jmx-service.sh` on
```

Comment:
Didn't we kill the JMX service?

Reply:
We sure did (I had learned that the [hard](https://github.com/scylladb/scylla-jmx/pull/240#issuecomment-2188574508) [way](https://github.com/scylladb/scylla-tools-java/pull/398#issuecomment-2188621454) :)).

However, this PR is not meant to refresh the entire document `docs/dev/docker-hub.md`; it only targets the stale description of `/proc/sys/fs/aio-max-nr`.